src/lycophron/app.py

```diff
@@ -110,14 +108,9 @@ def _init_project(self):
         """Initialize the project."""
         self.project.initialize()
 
-    def _init_logging(self):
-        """Initialize logging."""
-        init_logging(self.root_path)
-
     def recreate(self):
-        self.config.recreate()
         self.project.recreate()
-        self._remove_directory()
+        # self._remove_directory()
```

Comment:
```suggestion
```

We can remove it completely I guess, or make sure it only removes temporary dirs?